unitary/examples/quantum_chinese_chess/board.py

```diff
@@ -28,6 +28,20 @@
 # The default initial state of the game.
 _INITIAL_FEN = "RHEAKAEHR/9/1C5C1/P1P1P1P1P/9/9/p1p1p1p1p/1c5c1/9/rheakaehr w---1"
 
+# Constants for printing board
+_RESET = "\033[0m"
```

Comment:
Does all this work in colab?  I'm not sure how portable all these unix terminal codes are.  If it works in colab though, that's probably good enough

Reply:
Hi Doug, yes I just checked that it works in cola, with https://colab.research.google.com/drive/1B-7LmTN_gL4T1akVqUtoYO5dwzK_3IVK?usp=sharing, but replaced the files with this PR.